app/src/main/res/layout/content_item.xml

```diff
@@ -22,12 +22,12 @@
     app:explorationSplitViewMarginBottom="@{@dimen/space_0dp}"
     app:explorationSplitViewMarginEnd="@{@dimen/content_item_exploration_split_view_margin_end}"
     app:explorationSplitViewMarginStart="@{@dimen/content_item_exploration_split_view_margin_start}"
-    app:explorationSplitViewMarginTop="@{@dimen/content_item_exploration_split_view_margin_top}"
+    app:explorationSplitViewMarginTop="@{@dimen/space_0dp}"
     app:explorationViewMarginApplicable="@{viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
     app:explorationViewMarginBottom="@{@dimen/space_0dp}"
     app:explorationViewMarginEnd="@{@dimen/content_item_exploration_view_margin_end}"
     app:explorationViewMarginStart="@{@dimen/content_item_exploration_view_margin_start}"
-    app:explorationViewMarginTop="@{@dimen/content_item_exploration_view_margin_top}"
+    app:explorationViewMarginTop="@{@dimen/space_0dp}"
```

Comment:
Removed margin-top as this is getting controlled in `state_fragment` via `paddingTop`